### ISSUE TITLE
regex 2021.11.9 breaks on MacOS 11.6.1

### ISSUE THREAD

**Issue by tehranian** [mrabarnett/mrab-regex#444]
Howdy! Just a heads up that our `pip installs` started breaking, coincident with the release of `regex==2021.11.9`, two hours ago.

From our logs:
```
    Running setup.py install for regex ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/dtehranian/Documents/code/cloudhealth-flex-report/venv/bin/python3.9 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-install-xoxc6f5b/regex/setup.py'"'"'; __file__='"'"'/private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-install-xoxc6f5b/regex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-record-q4db5c7b/install-record.txt --single-version-externally-managed --compile --install-headers /Users/dtehranian/Documents/code/cloudhealth-flex-report/venv/include/site/python3.9/regex
         cwd: /private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-install-xoxc6f5b/regex/
    Complete output (126 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-11.3-x86_64-3.9
    creating build/lib.macosx-11.3-x86_64-3.9/regex
    copying regex_3/__init__.py -> build/lib.macosx-11.3-x86_64-3.9/regex
    copying regex_3/regex.py -> build/lib.macosx-11.3-x86_64-3.9/regex
    copying regex_3/_regex_core.py -> build/lib.macosx-11.3-x86_64-3.9/regex
    copying regex_3/test_regex.py -> build/lib.macosx-11.3-x86_64-3.9/regex
    running build_ext
    building 'regex._regex' extension
    creating build/temp.macosx-11.3-x86_64-3.9
    creating build/temp.macosx-11.3-x86_64-3.9/regex_3
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I/Users/dtehranian/Documents/code/cloudhealth-flex-report/venv/include -I/Users/dtehranian/.pyenv/versions/3.9.4/include/python3.9 -c regex_3/_regex.c -o build/temp.macosx-11.3-x86_64-3.9/regex_3/_regex.o
    regex_3/_regex.c:65:9: error: unknown type name 'RE_UINT32'
    typedef RE_UINT32 RE_CODE;
            ^
    regex_3/_regex.c:79:9: error: unknown type name 'RE_UINT32'
    typedef RE_UINT32 RE_STATUS_T;
            ^
    regex_3/_regex.c:246:5: error: type name requires a specifier or qualifier
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:246:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
               ^
    regex_3/_regex.c:248:5: error: type name requires a specifier or qualifier
        BOOL (*at_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:248:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:248:5: error: duplicate member 'BOOL'
        BOOL (*at_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:249:5: error: type name requires a specifier or qualifier
        BOOL (*at_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:249:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:249:5: error: duplicate member 'BOOL'
        BOOL (*at_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:250:5: error: type name requires a specifier or qualifier
        BOOL (*at_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:250:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:250:5: error: duplicate member 'BOOL'
        BOOL (*at_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:251:5: error: type name requires a specifier or qualifier
        BOOL (*at_default_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:251:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_default_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:251:5: error: duplicate member 'BOOL'
        BOOL (*at_default_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:252:5: error: type name requires a specifier or qualifier
        BOOL (*at_default_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:252:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_default_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:252:5: error: duplicate member 'BOOL'
        BOOL (*at_default_word_start)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:253:5: error: type name requires a specifier or qualifier
        BOOL (*at_default_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:253:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_default_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:253:5: error: duplicate member 'BOOL'
        BOOL (*at_default_word_end)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:254:5: error: type name requires a specifier or qualifier
        BOOL (*at_grapheme_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:254:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*at_grapheme_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
               ^
    regex_3/_regex.c:254:5: error: duplicate member 'BOOL'
        BOOL (*at_grapheme_boundary)(RE_StatePtr state, Py_ssize_t text_pos);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    regex_3/_regex.c:255:5: error: type name requires a specifier or qualifier
        BOOL (*is_line_sep)(Py_UCS4 ch);
        ^
    regex_3/_regex.c:255:12: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
        BOOL (*is_line_sep)(Py_UCS4 ch);
               ^
    regex_3/_regex.c:255:5: error: duplicate member 'BOOL'
        BOOL (*is_line_sep)(Py_UCS4 ch);
        ^
    regex_3/_regex.c:246:5: note: previous declaration is here
        BOOL (*has_property)(RE_LocaleInfo* locale_info, RE_CODE property, Py_UCS4
        ^
    fatal error: too many errors emitted, stopping now [-ferror-limit=]
    9 warnings and 20 errors generated.
    error: command '/usr/bin/clang' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/dtehranian/Documents/code/cloudhealth-flex-report/venv/bin/python3.9 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-install-xoxc6f5b/regex/setup.py'"'"'; __file__='"'"'/private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-install-xoxc6f5b/regex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/d8/spybd6cs33v201tvtl1q_prc0000gn/T/pip-record-q4db5c7b/install-record.txt --single-version-externally-managed --compile --install-headers /Users/dtehranian/Documents/code/cloudhealth-flex-report/venv/include/site/python3.9/regex Check the logs for full command output.
```

**Comment by fathom-parth** [comment]
Maybe related to
https://github.com/mrabarnett/mrab-regex/issues/443
which is now fixed in 2021.11.10

**Comment by mrabarnett** [comment]
Duplicate of issue #443.
